### ISSUE TITLE
Deprecation warning: use_x_sendfile will be removed in Flask 2.3

### ISSUE THREAD

**Issue by UlrichB22** [moinwiki/moin#1558]
Following message is issued by `src/moin/utils/_tests/test_send_file.py`:

`DeprecationWarning: 'use_x_sendfile' is deprecated and will be removed in Flask 2.3. Use 'USE_X_SENDFILE' in 'app.config' instead.`

send_file() fails with:

```
  File "/home/moint03/test/moin/src/moin/utils/send_file.py", line 154, in send_file
    if current_app.use_x_sendfile and filename:
AttributeError: 'Flask' object has no attribute 'use_x_sendfile'
```
Is there still a need to have our own copy of send_file() from [werkzeug/utils.py](https://github.com/pallets/werkzeug/blob/726eaa28593d859548da3477859c914732f012ef/src/werkzeug/utils.py#L318) ?

**Comment by ThomasWaldmann** [comment]
@UlrichB22 I fixed that, see my PR.

After that is reviewed / merged, we can try to upgrade to the latest flask/werkzeug releases.

About the need for send_file: it does some clever stuff in there. For many use cases, I guess anything would suffice, but (IIRC) back then when that was implemented, like e.g.:
- it was made in a way so it can scale up to ISO-Image sized files (on machines not having many GBs of RAM).
- it supports client-side caching, so stuff loads faster and generates less http requests to the wiki server.
